test/integration/resources/shoot-os.yaml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Why is this file here? We have templates in `hack/templates/resources` and examples, can we please reuse them?

Reply:
because, I stripped off the name and the domain from the resource example. The test generates those for you. That said, I could still remove it, and add a mention in the documentation.

Reply:
Please reuse whatever is possible, otherwise these things will run out of sync when we change something.